<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/ICONS/resources/textEditor</normaloff>:/ICONS/resources/textEditor</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionJoin_Server"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionDownload"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuHighlight">
     <property name="title">
      <string>Highlight</string>
     </property>
     <addaction name="actionPlain_Text"/>
     <addaction name="separator"/>
     <addaction name="actionC_C"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuHighlight"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionC_C_Compiler"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionJoin_Server"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionDownload"/>
   <addaction name="separator"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionReplace"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionC_C_Compiler"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/saveall</normaloff>:/ICONS/resources/saveall</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/download</normaloff>:/ICONS/resources/download</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/exit</normaloff>:/ICONS/resources/exit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionJoin_Server">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/connect</normaloff>:/ICONS/resources/connect</iconset>
   </property>
   <property name="text">
    <string>Join Server</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/disconnect</normaloff>:/ICONS/resources/disconnect</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/undo</normaloff>:/ICONS/resources/undo</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/redo</normaloff>:/ICONS/resources/redo</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/cut</normaloff>:/ICONS/resources/cut</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/copy</normaloff>:/ICONS/resources/copy</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/paste</normaloff>:/ICONS/resources/paste</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/search.png</normaloff>:/ICONS/resources/search.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/replace.png</normaloff>:/ICONS/resources/replace.png</iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionC_C_Compiler">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/c.png</normaloff>:/ICONS/resources/c.png</iconset>
   </property>
   <property name="text">
    <string>C/C++ Compiler</string>
   </property>
  </action>
  <action name="actionC_C">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/mark.png</normaloff>:/ICONS/resources/mark.png</iconset>
   </property>
   <property name="text">
    <string>C/C++</string>
   </property>
  </action>
  <action name="actionPlain_Text">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/ICONS/resources/mark.png</normaloff>:/ICONS/resources/mark.png</iconset>
   </property>
   <property name="text">
    <string>Plain Text</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
